packages/medusa/src/repositories/__mocks__/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./cart"
```

Comment:
I've only added the exports currently being used to the index file.
The idea would be that more can be added as they are needed.